api/python/provisioner/commands/check.py

```diff
@@ -455,79 +457,18 @@ def _bmc_accessibility(*, args: str) -> CheckEntry:
         :param args: bmc accessibility check specific parameters and arguments
         :return:
         """
-        res: CheckEntry = CheckEntry(cfg.Checks.BMC_ACCESSIBILITY.value)
-
-        # Check for import errors
-        if cortx_py_utils_import_error:
-            res.set_fail(checked_target=cfg.ALL_MINIONS,
-                         comment="Package cortx-py-utils not installed")
-            return res
-
-        try:
-            # Get node list
-            nodes = Check._get_pillar_data("cluster/node_list")
-            # Check if BMC is accessible for these nodes
-            BmcV().validate('accessible', nodes)
-        except Exception as exc:
-            res.set_fail(checked_target=cfg.ALL_MINIONS,
-                         comment=str(exc))
-        else:
-            res.set_passed(checked_target=cfg.ALL_MINIONS)
-
-        return res
+        return Check._validate_bmc_check('accessible')
```

Comment:
It is better to make `_bmc_accesibility` as ordinary class method (remove `@staticmethod`). So it will allow to refer to `_validate_bmc_check` using `self.` and be independent of class renaming:
```python
return self._validate_bmc_check('accessible')
```
Also, please, use constants for `accessible` and `stonith` strings.

Reply:
comment is addressed